<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waypoint Flight Planner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../WaypointPlannerResources.qrc">
    <normaloff>:/images/images/32x32/apps/spire.png</normaloff>:/images/images/32x32/apps/spire.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="WaypointMapView" name="mapView" native="true"/>
    </item>
    <item>
     <widget class="WaysetTimelineWidget" name="timelineView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Waypoint Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset_Flight"/>
   <addaction name="actionTest_Flight"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Mode"/>
   <addaction name="actionCreate_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionCoverage_Helper"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Problem"/>
   <addaction name="separator"/>
   <addaction name="actionImport_Solution"/>
   <addaction name="actionExport_Solution"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/48/48px-System-log-out.svg.png</normaloff>:/images/images/48/48px-System-log-out.svg.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSelect_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/editWaypointCursor.png</normaloff>:/images/images/editWaypointCursor.png</iconset>
   </property>
   <property name="text">
    <string>Edit Waypoints</string>
   </property>
  </action>
  <action name="actionCreate_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/createWaypointCursor.png</normaloff>:/images/images/createWaypointCursor.png</iconset>
   </property>
   <property name="text">
    <string>Create Waypoints</string>
   </property>
  </action>
  <action name="actionAuto_Fix">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/fixIcon.png</normaloff>:/images/images/fixIcon.png</iconset>
   </property>
   <property name="text">
    <string>Auto-Fix</string>
   </property>
  </action>
  <action name="actionCoverage_Helper">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/coverageHelperIcon.png</normaloff>:/images/images/coverageHelperIcon.png</iconset>
   </property>
   <property name="text">
    <string>Coverage Helper</string>
   </property>
  </action>
  <action name="actionImport_Solution">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/48/48px-Document-open.svg.png</normaloff>:/images/images/48/48px-Document-open.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Solution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionExport_Solution">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/48/48px-Document-save.svg.png</normaloff>:/images/images/48/48px-Document-save.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export Solution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="actionReset_Flight">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/32x32/reset or undo.png</normaloff>:/images/images/32x32/reset or undo.png</iconset>
   </property>
   <property name="text">
    <string>Reset Flight</string>
   </property>
  </action>
  <action name="actionTest_Flight">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/48/48px-Media-playback-start.svg.png</normaloff>:/images/images/48/48px-Media-playback-start.svg.png</iconset>
   </property>
   <property name="text">
    <string>Test Flight</string>
   </property>
  </action>
  <action name="actionOpen_Problem">
   <property name="icon">
    <iconset resource="../WaypointPlannerResources.qrc">
     <normaloff>:/images/images/48/48px-Document-open.svg.png</normaloff>:/images/images/48/48px-Document-open.svg.png</iconset>
   </property>
   <property name="text">
    <string>Open Problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaypointMapView</class>
   <extends>QWidget</extends>
   <header location="global">gui/WaypointMapView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WaysetTimelineWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/WaysetTimelineWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../WaypointPlannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
